src/main/java/dk/kb/present/View.java

```diff
@@ -100,6 +102,12 @@ public View(YAML conf, String originOfCollection) {
                 throw new RuntimeException(e); // Wrap for stream use
             }
         }
+
+        try {
+            placeholderXml = Resolver.resolveUTF8String("placeholder.xml");
```

Comment:
As the content of `placeholder.xml` is tiny, maybe just set it directly as a String instead of storing it in a file? That way it also becomes more transparent what it is.